### PR TITLE
Improve cut glass edge texture.

### PR DESCRIPTION
**Pull request by fozolo**
Also preserves connected_glass feature.

### CODE REVIEW

**Comment by fozolo** [comment]
@h-v-smacker, @sofar 

This fixes this issues raised in #113

screenshot_20180303_205059

**Comment by h-v-smacker** [comment]
Outstanding, I'll close my PR in favor of this.

File: nodes.lua [hunk 1]
```diff
@@ -459,6 +459,12 @@ for name, def in pairs(nodes) do
 	def.tiles = def.tiles or {"moreblocks_" ..name.. ".png"}
 	minetest.register_node("moreblocks:" ..name, def)
 	minetest.register_alias(name, "moreblocks:" ..name)
+
+	-- Use the primary tile for all sides of cut glasslike nodes
+	if #def.tiles > 1 and def.drawtype and def.drawtype:find("glass") then
```
Comment (SmallJoker) [inline nodes.lua]: `(def.drawtype == "glasslike_framed" or def.drawtype == "glasslike_framed_optional")` to avoid false positives

**Comment by sofar** [comment]
Please fix the small updates requested so we can merge this.

**Comment by Calinou** [comment]
Thanks! :smiley:

**Pull request merged**